src/Winton.AspNetCore.Seo/Winton.AspNetCore.Seo.csproj

```diff
@@ -2,17 +2,21 @@
   
   <PropertyGroup>
     <Authors>Winton</Authors>
+    <Company>Winton</Company>
     <Copyright>Copyright 2017 Winton</Copyright>
     <Description>ASP.NET Core Library for adding SEO metadata to websites</Description>
     <GenerateDocumentationFile>True</GenerateDocumentationFile>
     <GetVersion>False</GetVersion>
     <NoWarn>$(NoWarn);SA1101;SA1309;SA1413</NoWarn>
     <PackageId>Winton.AspNetCore.Seo</PackageId>
-    <PackageRequireLicenseAcceptance>False</PackageRequireLicenseAcceptance>
+    <PackageIconUrl>https://raw.githubusercontent.com/wintoncode/Winton.AspNetCore.Seo/master/icon.jpg</PackageIconUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/wintoncode/Winton.AspNetCore.Seo/master/LICENSE</PackageLicenseUrl>
+    <PackageProjectUrl>https://github.com/wintoncode/Winton.AspNetCore.Seo</PackageProjectUrl>
+    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
     <PackageTags>Winton, ASP.NET, Core, SEO, Search, Engine, Optimization</PackageTags>
     <PackageVersion>$(NuGetVersion)</PackageVersion>
     <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/wintoncode/Winton.AspNetCore.Seo</RepositoryUrl>
+    <RepositoryUrl>https://github.com/wintoncode/Winton.AspNetCore.Seo.git</RepositoryUrl>
```

Comment:
Does this need the .git at the end?

Reply:
It doesn't if it's just for use in the browser. Although with the `.git` extension I believe it can also be used to clone the repo. Again, not got a strong opinion on this one, I changed it to follow the convention of the other C# libs we have.

Reply:
Hmm, I think the .git just an alternative for legacy compatibility that GitHub provides. `git clone https://github.com/wintoncode/Winton.AspNetCore.Seo` just works.

Don't really mind though.